lib/settings/presentationSettings.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+var hooks = require('../hooks/hooks.js');
+
+
+var Promise     = require("bluebird")  
+  , coroutine   = Promise.coroutine
+  , fs = require("fs");
+
+var Exercise = db.model('Exercise');
+var Slideshow = db.model('Slideshow');
+var Setting = db.model('Setting');
+var ObjectId = require('mongoose').Types.ObjectId;
+
+Promise.promisifyAll(fs);
+
+
+module.exports = {
+
+    // TODO: need this?
+    getDustifySettingsOfExercisesAll: coroutine(function* getDustifySettingsOfExercisesAll(slideshowId) {
```

Comment:
`getDustifySettingsOfExercisesAll` sounds a bit strange because it has two verbs: get and dustify. Maybe you want to say `getDustSettingsofAllExercises`?